markdown.ipynb

```diff
@@ -4,21 +4,22 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `markdown.ipynb` Cell 1 Line 24_
```markdown
    <tr>
        <td>Foo</td>
        <td>Bar</td>
        <td>Baz</td>
    </tr>
</table>

I get 10 times more traffic from [Google] [1] than from
```
get 10 times more

_Respond and view the context [here](http://localhost:3000/smith-kyle/notebook-diff-tests/pull/24)._